web/templates/settings/admin_human_form.hbs

```diff
@@ -27,9 +27,9 @@
         <div class="custom-profile-field-form"></div>
     </form>
     <div class="input-group new-style">
-        <span {{#if owner_is_only_user_in_organization}}class="deactivate_user_button_tooltip"{{/if}}>
-            <button class="button rounded btn-danger deactivate_user_button" {{#if owner_is_only_user_in_organization}}disabled="disabled"{{/if}}>
-                {{t 'Deactivate user' }}
+        <span {{#if owner_is_only_user_in_organization}}class="deactivate_or_reactivate_user_button_tooltip"{{/if}}>
+            <button class="button rounded btn-danger deactivate_or_reactivate_user_button" {{#if owner_is_only_user_in_organization}}disabled="disabled"{{/if}}>
+                {{action}}
```

Comment:
Would it be better to instead have a separate button for reactivation and show one of them accordingly? This way we would be able to have a different click handler for reactivation button instead of having if-else block in the same handler.